.travis.yml

```diff
@@ -5,21 +5,25 @@
 # https://github.com/ros-infrastructure/ros_buildfarm/blob/master/doc/jobs/prerelease_jobs.rst
 # while this doesn't require sudo we don't want to run within a Docker container
 sudo: true
-dist: trusty
+dist: bionic
 language: python
-python:
-  - "3.4"
 env:
   global:
     - JOB_PATH=/tmp/devel_job
     - ABORT_ON_TEST_FAILURE=1
+    - INDEX_URL=https://raw.githubusercontent.com/ros-infrastructure/ros_buildfarm_config/production/index.yaml
   matrix:
+    - CHECK_PYTHON3_COMPILE=true
+    - ROS_DISTRO_NAME=indigo  OS_NAME=ubuntu OS_CODE_NAME=trusty ARCH=amd64 INDEX_URL=https://raw.githubusercontent.com/ros-infrastructure/ros_buildfarm_config/6a93d17/index.yaml
```

Comment:
Why bother to keep indigo in here at all?
I would propose to drop it altogether from CI (throughout the other repositories as well).

Reply:
@v4hn we have indigo test, but it is actually failing, -> https://travis-ci.com/github/PR2/linux_networking/builds/218549093
This means CI goes green even if indigo fails, but if there is someone who still uses indigo, they could know what is wrong with it.